plugins/defaultplugins/publish.go

```diff
@@ -52,23 +52,37 @@ func (plugin *Plugin) publishIfStateEvents(ctx context.Context) {
 	plugin.wg.Add(1)
 	defer plugin.wg.Done()
 
+	// store last errors to prevent repeating
+	var lastPublishErr error
+	var lastNotifErr error
+
 	for {
 		select {
 		case ifState := <-plugin.ifStateChan:
 			key := intf.InterfaceStateKey(ifState.State.Name)
 
 			if plugin.PublishStatistics != nil {
-				err := plugin.PublishStatistics.Put(key, ifState.State)
-				if err != nil {
-					plugin.Log.Error(err)
+				if err := plugin.PublishStatistics.Put(key, ifState.State); err != nil {
+					if err.Error() != lastPublishErr.Error() {
+						plugin.Log.Error(err)
+						lastPublishErr = err
```

Comment:
This can be simplified by setting `lastPublishErr` only once after the if clause with `Put`.